<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThrdBinTreeWidget</class>
 <widget class="QWidget" name="ThrdBinTreeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="15">
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="button_hideThrd">
     <property name="text">
      <string>隐藏线索化</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="button_postorder">
     <property name="text">
      <string>后序遍历</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QPushButton" name="button_levelorder">
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;button_levelorder&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>层序遍历</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="button_inThrd">
     <property name="text">
      <string>中序线索化</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QLabel" name="label_nodeCount">
     <property name="text">
      <string>   二叉树结点数:1</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="button_postThrd">
     <property name="text">
      <string>后序线索化</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="button_inorder">
     <property name="text">
      <string>中序遍历</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="1">
    <widget class="QLabel" name="label_treeDepth">
     <property name="text">
      <string>   二叉树深度:1</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="button_preThrd">
     <property name="text">
      <string>先序线索化</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>二叉树可视化</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="button_preorder">
     <property name="text">
      <string>先序遍历</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="button_levelThrd">
     <property name="text">
      <string>层序线索化</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
